src/mitm/ec_ip6nd_poison.c

```diff
@@ -90,7 +90,7 @@ static int ndp_poison_start(char *args)
       SAFE_FREE(h);
    }
 
-   if(EC_GBL_OPTIONS->silent) {
+   if(EC_GBL_OPTIONS->silent && !EC_GBL_OPTIONS->load_hosts) {
```

Comment:
What is the background of this change?

Reply:
it was a cherry-pick from master branch! removed and rebased now